### PR TITLE
Implement `get_ctrl_system` for swap bloqs

### PR DESCRIPTION
**Pull request by anurudhp**

### CODE REVIEW

File: qualtran/bloqs/basic_gates/swap.py [hunk 1]
```diff
@@ -103,6 +115,21 @@ def on_classical_vals(
     def adjoint(self) -> 'Bloq':
         return self
 
+    def get_ctrl_system(self, ctrl_spec: 'CtrlSpec') -> tuple['Bloq', 'AddControlledT']:
+        if ctrl_spec != CtrlSpec():
+            return super().get_ctrl_system(ctrl_spec=ctrl_spec)
+
+        cbloq = TwoBitCSwap()
```
Comment (mpharrigan) [inline qualtran/bloqs/basic_gates/swap.py:122]: please reserve this variable name for composite bloqs (sorry). `ctrl_bloq` if you need it to be agnostic or (preferred) `cswap`

File: qualtran/bloqs/basic_gates/swap.py [hunk 2]
```diff
@@ -243,6 +270,21 @@ def wire_symbol(self, reg: Optional['Register'], idx: Tuple[int, ...] = ()) -> '
     def adjoint(self) -> 'Bloq':
         return self
 
+    def get_ctrl_system(self, ctrl_spec: 'CtrlSpec') -> tuple['Bloq', 'AddControlledT']:
+        if ctrl_spec != CtrlSpec():
+            return super().get_ctrl_system(ctrl_spec=ctrl_spec)
+
+        cbloq = CSwap(self.bitsize)
```
Comment (mpharrigan) [inline qualtran/bloqs/basic_gates/swap.py:277]: `cswap`

**Review by mpharrigan — APPROVED** [review]
Thank you! Approving to not block this, but  please make the variable name changes I've suggested

File: qualtran/bloqs/basic_gates/swap.py (hunk 1 above)
Comment (anurudhp) [reply qualtran/bloqs/basic_gates/swap.py]: renamed to `cswap`

File: qualtran/bloqs/basic_gates/swap.py (hunk 2 above)
Comment (anurudhp) [reply qualtran/bloqs/basic_gates/swap.py]: renamed to `cswap`

**Pull request merged**